packages/angular_devkit/build_webpack/builders.json

```diff
@@ -2,11 +2,19 @@
   "$schema": "../architect/src/builders-schema.json",
   "builders": {
     "webpack": {
+      "implementation": "./src/webpack/index2",
       "class": "./src/webpack",
       "schema": "./src/webpack/schema.json",
       "description": "Build a webpack app."
     },
+    "dev-server": {
```

Comment:
why 2 builders that do the same thing?

Reply:
I was thinking of removing the other one. `build-webpack:webpack-dev-server` is redundant. Same for `build-webpack:webpack`, was thinking of renaming it `build`.

Reply:
I think it makes sense